app/helpers/navbar_helper.rb

```diff
@@ -2,4 +2,12 @@ module NavbarHelper
   def active_class(link_path)
     current_page?(link_path) ? "active" : ""
   end
+
+  def settings_active_class(project)
```

Comment:
I am not sure about how I can approach this problem. Do you have any suggestions? 
I could only think of using a single page containing the current pages as partials in a tab-like container.